support-services/build.sbt

```diff
@@ -6,7 +6,6 @@ description := "Scala library to provide shared services to Guardian Support pro
 
 libraryDependencies ++= Seq(
   "software.amazon.awssdk" % "dynamodb" % awsClientVersion2,
-  "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.1",
```

Comment:
no longer needed and incompatible with deps

Reply:
nice well done for deleting it